common/changes/@snowplow/browser-tracker/issue-1248-send-failure-retry-option_2023-10-20-16-06.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@snowplow/browser-tracker",
+      "comment": "Retry control option",
```

Comment:
Is this change item here intentional?

Reply:
Yeah, I had to make one since the API docs have changed. I've re-made the change file here with a more useful comment:
https://github.com/snowplow/snowplow-javascript-tracker/pull/1257/files#diff-69080c0b84c22e16c271e7e96183dcb2376f24014b73674c13101d66013f7f48R5